src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.html.twig

```diff
@@ -0,0 +1,316 @@
+
+{% if login_box is defined and login_box is not empty %}
+  {% set _login_box = _login_box|merge(login_box) %}
+{% endif %}
+
+{% if language_selector is defined and language_selector is not empty %}
+  {% set _language_selector = _language_selector|merge(language_selector) %}
+{% endif %}
+
+{% if search_toggle is defined and search_toggle is not empty %}
+  {% set _search_toggle = _search_toggle|merge(search_toggle) %}
+{% endif %}
+
+{% if search_form.text_input is defined and search_form.text_input is not empty %}
+  {% set _search_form = _search_form|merge({ text_input: _search_form.text_input|merge(search_form.text_input) }) %}
+{% endif %}
+
+{% if search_form.button is defined and search_form.button is not empty %}
+  {% set _search_form = _search_form|merge({ button: _search_form.button|merge(search_form.button) }) %}
+{% endif %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<header class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <div class="ecl-site-header-harmonised__container ecl-container">
+    <div class="ecl-site-header-harmonised__top">
+      <a
+        class="ecl-link ecl-link--standalone ecl-site-header-harmonised__logo-link"
+        href="{{ _logo.href }}"
+        aria-label="{{ _logo.title }}"
+      >
+        <img
+          alt="{{ _logo.alt }}"
+          title="{{ _logo.title }}"
+          class="ecl-site-header-harmonised__logo-image"
+          src="{{ _logo.src }}"
+        />
+      </a>
+      <div class="ecl-site-header-harmonised__action">
+      {% if _login_toggle is not empty and _login_box is not empty %}
+        <div class="ecl-site-header-harmonised__login-container">
+        {% if _logged %}
+          <a
+            class="ecl-link ecl-link--standalone ecl-site-header-harmonised__login-toggle"
+            href="{{ _login_box.href_logged }}"
+            aria-controls={{ _login_box.id }}
+            aria-expanded="false"
+          >
+            {% include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+              icon: {
+                path: _icon_file_path,
+                type: 'general',
+                name: 'logged-in',
+                size: 's'
+              }
+            } only %}
+             {{- login_toggle.label_logged -}}
+          </a>
+          <div
+            id="{{ _login_box.id }}"
+            class="ecl-site-header-harmonised__login-box"
+            data-ecl-login-box
```

Comment:
data-ecl-login-box="true"?

Reply:
same for data-ecl-language-list-overlay="true" in language-list component.

Reply:
Same as in the other pr, check the html tab, in the test we get these booleans as attribute_name="" which is not nice, though...